manual/src/threelang/web3gw/client.md

```diff
@@ -0,0 +1,32 @@
+# Client
+An actor that loads all the needed clients to interact with the supported chains (`tfchain`, `stellar`, `etherum`, `bitcoin`, ...).
+
+## Actions
+- action name: `!!web3gw.client.load`
```

Comment:
Why do we need this? Let's move all keys to web3gw.keys.define so that we have something like this:
```
!!web3gw.keys.define
       network:main
       tfgrid:mnemoniccomeshere
       stellar:stellarkeycomeshere
       eth:ethkeycomeshere
       btc:btckeycomeshere
```
the network doesn't have to be provided, the default value is main. That way we avoid the actions on every client to load the key.

Reply:
by `!!web3gw.client.load` i meant loading all the client needed based on the provided params. this action will be executed in the beginning loading all the clients so the following actions can use these clients. renamed it to `!!web3gw.keys.define`

edited the default values to be `main` for tfchain and `public` for stellar. but left the option so if any one want to use the module with different networks. is it okay?